cartridges/int_bolt_sfra/cartridge/controllers/Order.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var server = require('server');
+var Order = module.superModule;
+server.extend(Order);
+
+var Resource = require('dw/web/Resource');
+var URLUtils = require('dw/web/URLUtils');
+var Site = require('dw/system/Site');
+var OrderMgr = require('dw/order/OrderMgr');
+var Locale = require('dw/util/Locale');
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+var reportingUrlsHelper = require('*/cartridge/scripts/reportingUrls');
+var OrderModel = require('*/cartridge/models/order');
+var userLoggedIn = require('*/cartridge/scripts/middleware/userLoggedIn');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+
+/**
+ * Exclude order customer ID check for Bolt SSO, since order is assigned to new created account 
+ * if shopper creates account during checkout.
+ */
+server.replace('Confirm',
+    consentTracking.consent,
+    server.middleware.https,
+    csrfProtection.generateToken,
+    function (req, res, next) {    
+    var order;
+    var boltEnableSSO = Site.getCurrent().getCustomPreferenceValue('boltEnableSSO');
+
+    if (!req.form.orderToken || !req.form.orderID) {
+        res.render('/error', {
+            message: Resource.msg('error.confirmation.error', 'confirmation', null)
+        });
+        return next();
+    }
+
+    order = OrderMgr.getOrder(req.form.orderID, req.form.orderToken);
+
+    if (!order || (!boltEnableSSO && order.customer.ID !== req.currentCustomer.raw.ID)) {
```

Comment:
this line is different from base cartridge.
I tried saving the original guest customer id to the session or order and check the guest customer id here, but the raw session ID and `req.currentCustomer.raw.ID` in `Login-OAuthRedirectBolt` request and in `Order-Confirm` request are different.

Reply:
Can you add this comment in the PR to the code so the user knows what is different from the base cartridge.